packages/runtime/src/reflection-contains-message-type.ts

```diff
@@ -0,0 +1,22 @@
+import {IMessageType, MESSAGE_TYPE} from './message-type-contract';
+
+/**
+ * The interface that models storing type in a symbol property.
+ */
+export interface MessageTypeContainer<T extends object> {
+    [MESSAGE_TYPE]: IMessageType<T>;
+}
+
+/**
+ * Check if the provided object is a proto message.
+ */
+export function containsMessageType<T extends object>(msg: object): msg is MessageTypeContainer<T> {
+    return (msg as MessageTypeContainer<T>)[MESSAGE_TYPE] != null;
+}
+
+/**
+ * Get a message type from a proto message.
+ */
+export function getMessageType<T extends object>(msg: MessageTypeContainer<T>): IMessageType<T> {
+    return msg[MESSAGE_TYPE];
+}
```

Comment:
```suggestion
```

With the typing change above, I don't see any benefit from having this function.

You could easily express this with

```ts
import {containsMessageType, MESSAGE_TYPE} from "@protobuf-ts/runtime";
const type = containsMessageType(msg) ? msg[MESSAGE_TYPE] : undefined;
```

Or

```ts
if (containsMessageType(msg)) {
   msg[MESSAGE_TYPE].toJson(msg)
} else {
   throw "cannot redact"
}
```

Or 

````ts
assert(containsMessageType(msg), "cannot redact");
msg[MESSAGE_TYPE].toJson(msg)
```

Would you agree?